--- /etc/nginx/sites-available/default	2023-05-30 19:31:46.000000000 +0200
+++ /etc/nginx/sites-available/default	2023-03-11 00:25:15.443292562 +0100
@@ -19,13 +19,19 @@
 # Default server configuration
 #
 server {
-	listen 80 default_server;
-	listen [::]:80 default_server;
+	#listen 80 default_server;
+	#listen [::]:80 default_server;
 
 	# SSL configuration
 	#
-	# listen 443 ssl default_server;
-	# listen [::]:443 ssl default_server;
+	listen 443 ssl default_server;
+	listen [::]:443 ssl default_server;
+
+	ssl_certificate     /app/cert/server.crt;
+	ssl_certificate_key /app/cert/server.key;
+	ssl_protocols       TLSv1 TLSv1.1 TLSv1.2;
+	ssl_ciphers         HIGH:!aNULL:!MD5;
+
 	#
 	# Note: You should disable gzip for SSL traffic.
 	# See: https://bugs.debian.org/773332
@@ -48,7 +54,8 @@
 	location / {
 		# First attempt to serve request as file, then
 		# as directory, then fall back to displaying a 404.
-		try_files $uri $uri/ =404;
+		#try_files $uri $uri/ =404;
+		proxy_pass http://127.0.0.1:8000;
 	}
 
 	# pass PHP scripts to FastCGI server
